cmd/terramate/cli/cloud.go

```diff
@@ -1108,7 +1120,7 @@ func (c *cli) newBitbucketReviewRequest(pr *bitbucket.PR) *cloud.ReviewRequest {
 		Author: cloud.Author{
 			ID:        pr.Author.UUID,
 			Login:     pr.Author.DisplayName,
-			AvatarURL: avatarURL,
+			AvatarURL: pr.Author.Links.Avatar.Href,
```

Comment:
wouldn't panic in case of Links or Avatar attribute being nil?

Reply:
I changed the fields to be values instead of pointers.